visualizations/frontend/filters/v2/TRangeInput.vue

```diff
@@ -76,14 +77,20 @@
         },
         methods: {
             update(e) {
-                if (e.minValue === e.min && e.maxValue == e.max) {
-                    this.unsetFilterValue('min', true);
-                    this.unsetFilterValue('max', true);
-                } else {
-                    this.barMinValue = e.minValue;
-                    this.barMaxValue = e.maxValue;
+                if (this.throttleId) {
+                    clearTimeout(this.throttleId);
                 }
-            }
+
+                this.throttleId = setTimeout(() => {
+                    if (e.minValue === e.min && e.maxValue == e.max) {
+                        this.unsetFilterValue('min', true);
+                        this.unsetFilterValue('max', true);
```

Comment:
```suggestion
                        this.unsetFilterValue('min');
                        this.unsetFilterValue('max');
```

Correct me if I'm wrong, but I think that there is no longer a second argument on `unsetFilterValue`: https://github.com/snyk/topcoat-core/blob/a0eb0bf6a76bed8c7df0efe026700d4c10236b40/frontend/src/mixins/visualization_mixin.js#L301

Reply:
You're right. That's part of the old code.